README.md

```diff
@@ -1,10 +1,11 @@
 # Ruby Project
 
+[Be a Ruby](https://www.bearuby.org/) is a local non-profit organization that distributes food and clothing donations to people in need in the Grand Rapids community. During their events, items such as take-out boxes and cups are stored in a trailer and removed by volunteers as they are needed. Currently, they have no system to track when these items are removed from the trailer, which makes it difficult to know when they are running out of a particular item. Our mission is to create Ruby, a custom mobile application, that will provide an intuitive interface for them to update and track their trailer inventories.
```

Comment:
2nd sentence makes it seem that items are only stored in the trailer during events.
Update: "During their events, volunteers remove stored items like take-out boxes and cups as they are needed.

Reply:
I wanted to introduce the trailer here so that it makes sense in the rest of the statement. Are you saying that it makes it sound like items are not in the trailer between events, or that not all items are in the trailer during events? I'm not sure if they store items in the trailer between events, but I do think they are mostly focused on what is removed from the trailer (vs from other places)

Reply:
I was saying that items would probably always be in the trailer, and they would be removed at events. It works both ways, so I will figure out how to approve it.